tests/engine_test.py

```
@@ -862,3 +862,440 @@ def test_validate_with_scoped_breaking_change(self):
+        )
+        validator = CommitMessageValidator(rule)
+        context = ValidationContext(stdin_text="feat: add new feature")
+
+        result = validator.validate(context)
+        assert result == ValidationResult.PASS
+
+    @patch("commit_check.engine.get_commit_info")
+    @patch("commit_check.engine.has_commits")
+    @pytest.mark.benchmark
+    def test_commit_message_validator_failure(
+        self, mock_has_commits, mock_get_commit_info
```

Ruff (0.14.8)

Unused method argument: `mock_get_commit_info`

(ARG002)